docs/current/162770-faq.md

```diff
@@ -2,12 +2,142 @@
 
+### How do I install Dagger?
+
+Refer to the documentation for information on how to install the [Dagger CLI](./cli/465058-install.md) and the Dagger [Go](./sdk/go/371491-install.md), [Node.js](./sdk/nodejs/835948-install.md) and [Python](./sdk/python/866944-install.md) SDKs.
+
+### How do I update Dagger?
+
+:::tip
+[Learn more about compatibility between the Dagger Engine, the Dagger CLI and Dagger SDKs](#what-compatibility-is-there-between-the-dagger-engine-dagger-sdks-and-dagger-cli-versions).
+:::
+
+#### CLI
+
+To update the Dagger CLI, download and uncompress the [latest release of the Dagger CLI](https://github.com/dagger/dagger/releases) for your platform (replace the placeholders with the correct values) and replace your existing Dagger CLI binary with the newer version:
+
+```shell
+curl -L https://dl.dagger.io/dagger/install.sh | sh
+```
+
+Homebrew users can alternatively use the following commands:
+
+```shell
+brew update
+brew upgrade dagger
+```
+
+#### SDKs
+
+To update a Dagger SDK, follow the same procedure that you would follow to update any other SDK package in your chosen development environment. For example:
+
+- Go SDK
+
+  ```shell
+  go get -u dagger.io/dagger
+  ```
+
+- Node.js SDK
+
+  <Tabs>
+  <TabItem value="npm">
+
+  ```shell
+  npm install @dagger.io/dagger@latest --save-dev
+  ```
+
+  </TabItem>
+  <TabItem value="yarn">
+
+  ```shell
+  yarn upgrade --latest @dagger.io/dagger --dev
+  ```
+
+  </TabItem>
+  </Tabs>
+
+- Python SDK
+
+  <Tabs>
+  <TabItem value="pip">
+
+  ```shell
+  pip install -U dagger-io
+  ```
+
+  </TabItem>
+  <TabItem value="conda">
+
+  ```shell
+  conda update dagger-io
+  ```
+
+  </TabItem>
+  </Tabs>
+
+### What compatibility is there between the Dagger Engine, Dagger SDKs and Dagger CLI versions?
+
+- The Dagger CLI is released in tandem with the Dagger Engine and thus shares a version number with it.
+- Dagger SDKs automatically provision a Dagger Engine at a compatible version.
+
+Each release notes the compatible Dagger Engine version in its release notes. If running from the CLI, or providing a CLI for the SDK to use via the `_EXPERIMENTAL_DAGGER_CLI_BIN` variable, check the release notes of the SDK, which indicate the required CLI and Engine versions.
+
+The image below shows an example of the compatibility information available in the release notes:
+
+![Release information](/img/current/faq/release-notes.png)
+
+### How do I uninstall Dagger?
+
+Follow these steps:
+
+1. Remove the Dagger CLI using the following command:
+
+  ```shell
+  sudo rm /usr/local/bin/dagger
+  ```
+
+  Homebrew users can alternatively use the following command:
+
+  ```shell
+  brew uninstall dagger
+  ```
+
+1. Remove the `dagger` sub-directory of your `$XDG_CACHE_HOME` and `$XDG_CONFIG_HOME` directories. These are Dagger cache locations containing the auto-provisioned Dagger CLI (used for SDK operations).
```

Comment:
They aren't both caches, the CLI is downloaded only to `$XDG_CACHE_HOME` but there's other things there (oci blobs, progrock socket...), while `$XDG_CONFIG_HOME` can for example store credentials from dagger login/logout (cloud). None of these are SDK specific, they're used by the CLI and SDKs. 

No need to enumerate these things as they are subject to change. Pointing out where the CLI is downloaded to by the SDK can be useful though, but it should be clearer that it's just an example and that applies only to XDG_CACHE_HOME.

Reply:
**There should also be a note** that the paths below can be different. These are **just defaults**.

At some point we should have an easy way for the user to know which directories are actually used, for example:
- **CLI and Go SDK:** `dagger info`
- **Python SDK**: `python -c "import dagger; print(dagger.info())"`

For now we can add links to the various third-party packages that we use to get these paths. I've listed them in https://github.com/dagger/dagger/issues/3963#issuecomment-1369296005:

- **Go** (SDK and CLI): https://github.com/adrg/xdg
- **NodeJS**: https://github.com/sindresorhus/env-paths
- **Python**: https://github.com/platformdirs/platformdirs

Reply:
And a third thing: Windows is missing here. The path for windows is in those libraries I listed. We can add it later though.

Reply:
Added